dropshot/tests/fail/bad_channel13.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 Oxide Computer Company
+
+// XXX: The error message produced by this test isn't great -- we should fix it.
```

Comment:
what do we not like? the blather after the first section (or did I write this comment?)